www/includes/easyparliament/page.php

```diff
@@ -1733,60 +1733,6 @@ public function glossary_search_form($args) {
             $URL->remove(array('g'));
             $type = "<input type=\"hidden\" name=\"type\" value=\"2\">";
         }
-
-        $add_link = $URL->generate('url');
-        ?>
-        <form action="<?php echo $add_link; ?>" method="get">
-        <?php echo $type; ?>
-        <p>Help make TheyWorkForYou.com better by adding a definition:<br>
-        <label for="g"><input type="text" name="g" value="<?php echo $formcontent; ?>" size="45">
-        <input type="submit" value="Search" class="submit"></label>
-        </p>
-        </form>
```

Comment:
Looks like the whole `glossary_search_form` function can be removed, it's now not doing anything but setting some variables that aren't used.